plugins.js

```diff
@@ -0,0 +1,51 @@
+module.exports = [
+  {
+    name: 'postcss-simple-vars',
+  },
+  {
+    name: 'postcss-advanced-variables',
+    config: { disable: ['@import'] },
+    url: 'https://github.com/jonathantneal/postcss-advanced-variables',
+  },
+  {
+    name: 'postcss-mixins',
+  },
+  {
+    name: 'postcss-nesting',
+  },
+  {
+    name: 'postcss-nested',
+  },
+  {
+    name: 'short',
+    pkgName: 'postcss-short',
```

Comment:
I'd personally prefer calling it `postcss-short`. The repo, website, and npm package use `postcss-short`, but the readme uses `short`.

@jonathantneal What's the correct name for displaying it on the REPL?

Reply:
name is used to match with postcss-plugins data. We can always have a third property preferredName, but I think it should be the same than the one displayed on postcss.parts website, don't you think ?

Reply:
OK, if it's that way on postcss-parts, then leave it that way.

Reply:
Yes, I should update the README to be `postcss-short`. Amazing low priority unless someone wants to file an issue. 😄

Reply:
PR done: https://github.com/jonathantneal/postcss-short/pull/14